File: src/org/onyxplatform/api/java/Lifecycles.java
package org.onyxplatform.api.java;

public class Lifecycles 
	extends OnyxVector
{
    public Lifecycles() {
	    super();
    }
    
    private Lifecycles(Lifecycles ls) {
	    super( ls.vContents );
    }
    
    public void addCall(Lifecycle cs) {
	    super.vContents = super.vContents.cons( cs );
    }
}

File: test/onyx_java/test/single_javafn/lifecycles.clj
(ns onyx-java.test.single-javafn.lifecycles
  (:require [clojure.test :refer [deftest is]]) 
  (:import [org.onyxplatform.api.java Lifecycles]
           [org.onyxplatform.api.java.utils AsyncLifecycles MapFns]))

(defn build-lifecycles []
  
  )


(deftest valid-lifecycles?
  (let [cat (build-lifecycles)]
    (is true)))

